src/main/java/com/lorepo/icf/utils/ExtendedRequestBuilder.java

```diff
@@ -11,7 +11,7 @@ public class ExtendedRequestBuilder extends RequestBuilder {
 	
 	private static boolean withCredentials = false;
 	private static String signingPrefix = null;
-	private static List<String> whitelist = new ArrayList(Arrays.asList("mauthor", "lorepocorporate"));
+	private static List<String> whitelist = new ArrayList();
```

Comment:
lista początkowa została wyczyszczona, prosze o wyjaśnienie dlaczego, tym bardziej, że w drugiej PR nie widzę wywołania metody która, dodawałaby te pozycje

Reply:
Lista została wyczyszczona w celu zwiększania bezpieczeństwa. Pierwotnie uważałem to za dobry pomysł jednak po przemyśleniu tak zdecydowałem gdyż:
* Wymusza to że domeny takie jak https://evil.mauthor.com będą akceptowalne domyślnie.
* Andrzej mnie poinformował, że nie ma planów aby mA zaczął korzystać z zamkniętego CDN, A skoro nie ma wobec tego planów to na mA jest to zbędne

Reply:
Jeżeli coś w tym temacie się zmieni to i tak po stronie mA będzie trzeba użyć innych komend playerowych